ssz_serialization/merkleization.nim

```diff
@@ -668,6 +659,17 @@ template mergedHash(x: HashList|HashArray, vIdxParam: int64): Digest =
       hashTreeRootCached(x, vIdx),
       hashTreeRootCached(x, vIdx + 1))
 
+func hashTreeRootCached*(x: HashArray, vIdx: int): Digest =
```

Comment:
@zah Shouldn't this be `int64` instead of `int`? I left it as is for now, but it seems weird to have different types here.

Reply:
it should be, yes

Reply:
OK: https://github.com/status-im/nim-ssz-serialization/pull/11